parsers/shared.pegjs

```diff
@@ -17,12 +17,15 @@ ident "identifier"
 reference "reference"
     = "global(" _ ref:ident _ ")" { return { name : ref, global : true }; }
```

Comment:
This should be `= global_keyword "(" _ ...` now, and `global_keyword` should come before it.